mimic/test/dummy.py

```diff
@@ -93,36 +96,103 @@ def resource(self):
         return example_resource
 
 
-@implementer(IExternalAPIMock, IPlugin)
-class ExampleExternalAPI(object):
+@implementer(IEndpointTemplate)
+class ExampleEndpointTemplate(object):
     """
-    Example external API the return nothing
+    Example End-Point Template
```

Comment:
ATM, all endpoint templates are only for external services; but I'll note it accordingly in case we want to change that in the future for internally hosted services too.